api/krusty/validatingwebhook_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package krusty_test
+
+import (
+	"testing"
+
+	kusttest_test "sigs.k8s.io/kustomize/api/testutils/kusttest"
+)
+
+func TestValidatingWebhookCombinedNamespaces(t *testing.T) {
```

Comment:
Nicely done.  

Please just add a reference to the issue so that this test has some context, e.g.

```
// Reproduce issue #3732
```

Reply:
Thanks! Updated with comment.